### PR TITLE
create table test driven for message_google_iap_get_coins_test

### PR DESCRIPTION
**Pull request by levanhieuhust**
**What is the purpose of the change**

- Write table-driven test code

**Brief Changelog**

- Write all cli test cases with table-driven test for **ValidateBasic** function

**Testing and Verifying**

- This change is a trivial rework/code cleanup without any test coverage

**Documentation and Release Note**

- Does this pull request introduces a new feature or user-facing behaviour changes? no
- How is the feature or change documented? not applicable

### CODE REVIEW

File: x/pylons/types/message_google_iap_get_coins_test.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+				Creator:           correctAddr,
+				ProductId:         "free_pylons",
+				ReceiptDataBase64: encodedBase64Digest,
+				PurchaseToken:     "INVALID_PURCHASETOKEN",
+				Signature:         signaturebase64Encoded,
+			},
+			err: fmt.Errorf("purchaseToken does not match with receipt data"),
+		},
+		{
+			desc: "Valid",
```
Comment (mustafapylons) [inline x/pylons/types/message_google_iap_get_coins_test.go:76]: Shouldn't the description be "Invalid Product Id" instead of "Valid"?

**Review by mustafapylons — COMMENTED** [review]
One issue, otherwise code is good.

File: x/pylons/types/message_google_iap_get_coins_test.go (hunk 1 above)
Comment (levanhieuhust) [reply x/pylons/types/message_google_iap_get_coins_test.go]: Thank you! I have fix it in https://github.com/Pylons-tech/pylons/pull/940/commits/3fdd56878fc01ce4f7c398983c91a34443677114.

**Review by faddat — APPROVED** [review]

**Review by mustafapylons — APPROVED** [review]
Looks good to me.

**Pull request merged**